src/test/resources/pages/instructorFeedbackResultsSortGiverRecipientQuestionTeam.html

```diff
@@ -1939,7 +1939,7 @@
                                   [more]
                                 </a>
                                 <br>
-                                <span id="questionAdditionalInfo-8-section-1-giver-1-recipient-1" style="display: none;">
+                                <span id="questionAdditionalInfo-8-section-1-giver-1-recipient-1" style="display:none;">
```

Comment:
Probably should revert this change if possible - the missing space does not seem consistent with the rest of the file so I'm assuming it's unintentional, although if this is not possible (eg. it was caused by JS modifying the DOM and causes test failures if changed) then it should be fine to leave this change here.